app/src/main/res/values/strings.xml

```diff
@@ -571,4 +571,5 @@
   <string name="profile_edit_fragment_test_activity_label">Profile Edit Fragment Test Activity</string>
   <string name="administrator_controls_fragment_test_activity_label">Administrator Controls Fragment Test Activity</string>
   <string name="select_a_topic_to_start">Select a Topic to Start</string>
+  <string name="continue_studying">Continue Studying</string>
```

Comment:
This string name should describe what the button is/for rather than reflect its contents (as that's more useful for identifying it, helps with the fact that string names are global identifiers in the app, and future proofs the string against its label changing in the future).

Reply:
done